src/object/srv_obj.c

```diff
@@ -614,10 +741,15 @@ obj_local_rw(crt_rpc_t *rpc, struct ds_cont_hdl *cont_hdl,
 	uint64_t		time_start = 0;
 	struct obj_tls		*tls = obj_tls_get();
 	struct bio_desc		*biod;
+	struct daos_oclass_attr *oca = NULL;
+				 daos_oclass_attr_find(orw->orw_oid.id_pub);
```

Comment:
could you please remove L745? since you have to refresh the patch as confliction.